core/domain/stats_jobs_continuous.py

```diff
@@ -195,8 +195,8 @@ def reduce(key, stringified_values):
                 if latest_interaction_id != loaded_interaction_id and (
                         latest_version != exp.version):
                     yield (
-                        'Ignoring answers submitted to version %s and below '
-                        'since the latest exploration version is %s' % (
+                        'INFO: Ignoring answers submitted to version %s and '
```

Comment:
For parity, can you also add 'ERROR: ' prefixes to the other 3 outputs?

Reply:
Done.